packages/frames.js/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "frames.js",
-  "version": "0.8.2",
+  "version": "0.8.2-beta5",
```

```suggestion
  "version": "0.8.2-beta.5",
```
beta5 doesn't adhere to [semver](https://semver.org/#backusnaur-form-grammar-for-valid-semver-versions), kindly update it.